nbdime/diffing/generic.py

```diff
@@ -55,14 +60,22 @@ def compare_strings_approximate(x, y, threshold=0.7):
     # equal items, at least in the Myers diff algorithm.
     # Most other comparisons will likely not be very similar,
     # and the (real_)quick_ratio cutoffs will speed up those.
+
     # So the heavy ratio function is only used for close calls.
     # s = difflib.SequenceMatcher(lambda c: c in (" ", "\t"), x, y, autojunk=False)
     s = difflib.SequenceMatcher(None, x, y, autojunk=False)
+
+    # Use only the fast ratio approximations first
     if s.real_quick_ratio() < threshold:
         return False
     if s.quick_ratio() < threshold:
         return False
-    return s.ratio() > threshold
+
+    # Skip slower and stricter check unless if quick is set
+    if quick:
+        return True
```

Comment:
Should  be True. But I guess this was a bad idea. The quick ratios are too inaccurate to trust.